hs/src/Reach/Eval/Core.hs

```diff
@@ -3812,7 +3816,8 @@ evalId_ lab x = do
         SLM_ConsensusStep -> c
         SLM_ConsensusPure -> c
   let _lab' = lab <> " in " <> elab
-  infectWithId_sss x <$> (env_lookup (LC_RefFrom lab) x =<< env)
```

Comment:
I was referring to how you changed it from this single line into multiple lines

Reply:
the one-liner doesn't work now that `infectWithId_sss` is in the `App` monad

Reply:
Aaaahhh, `=<<` is what you want